workers/loc.api/sync/dao/db-migrations/sqlite-migrations/migration.v19.js

```diff
@@ -0,0 +1,353 @@
+        ON fundingCreditHistory(user_id, subUserId, mtsUpdate)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX positionsHistory_user_id_symbol_mtsUpdate
+        ON positionsHistory(user_id, symbol, mtsUpdate)`,
+      `CREATE INDEX positionsHistory_user_id_basePrice_mtsUpdate
+        ON positionsHistory(user_id, basePrice, mtsUpdate)`,
+      `CREATE INDEX positionsHistory_user_id_mtsUpdate_mtsCreate
+        ON positionsHistory(user_id, mtsUpdate, mtsCreate)`,
+      `CREATE INDEX positionsHistory_user_id_mtsUpdate
+        ON positionsHistory(user_id, mtsUpdate)`,
+      `CREATE INDEX positionsHistory_user_id_subUserId_mtsUpdate
+        ON positionsHistory(user_id, subUserId, mtsUpdate)
+        WHERE subUserId IS NOT NULL`,
+      `CREATE INDEX positionsHistory_subUserId_id
+        ON positionsHistory(subUserId, id)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX positionsSnapshot_user_id_mtsUpdate
+        ON positionsSnapshot(user_id, mtsUpdate)`,
+      `CREATE INDEX positionsSnapshot_user_id_symbol_mtsUpdate
+        ON positionsSnapshot(user_id, symbol, mtsUpdate)`,
+      `CREATE INDEX positionsSnapshot_user_id_subUserId_mtsUpdate
+        ON positionsSnapshot(user_id, subUserId, mtsUpdate)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX logins_user_id_ip_time
+        ON logins(user_id, ip, time)`,
+      `CREATE INDEX logins_user_id_id_time
+        ON logins(user_id, id, time)`,
+      `CREATE INDEX logins_user_id_time
+        ON logins(user_id, time)`,
+      `CREATE INDEX logins_user_id_subUserId_time
+        ON logins(user_id, subUserId, time)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX changeLogs_user_id_ip_mtsCreate
+        ON changeLogs(user_id, ip, mtsCreate)`,
+      `CREATE INDEX changeLogs_user_id_userAgent_mtsCreate
+        ON changeLogs(user_id, userAgent, mtsCreate)`,
+      `CREATE INDEX changeLogs_user_id_mtsCreate
+        ON changeLogs(user_id, mtsCreate)`,
+      `CREATE INDEX changeLogs_user_id_subUserId_mtsCreate
+        ON changeLogs(user_id, subUserId, mtsCreate)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX tickersHistory_symbol_bid_mtsUpdate
+        ON tickersHistory(symbol, bid, mtsUpdate)`,
+      `CREATE INDEX tickersHistory_symbol_ask_mtsUpdate
+        ON tickersHistory(symbol, ask, mtsUpdate)`,
+      `CREATE INDEX tickersHistory_symbol_mtsUpdate
+        ON tickersHistory(symbol, mtsUpdate)`,
+
+      `CREATE INDEX statusMessages_key__type_timestamp
+        ON statusMessages(key, _type, timestamp)`,
+      `CREATE INDEX statusMessages_key_price_timestamp
+        ON statusMessages(key, price, timestamp)`,
+      `CREATE INDEX statusMessages_key_priceSpot_timestamp
+        ON statusMessages(key, priceSpot, timestamp)`,
+      `CREATE INDEX statusMessages_key_timestamp
+        ON statusMessages(key, timestamp)`,
+
+      `CREATE INDEX candles__timeframe_mts
```

Comment:
actually we are not looking for _timeframe, _symbol, mts right?

Reply:
double checked and you are right we can use when converting currency. Would add `CREATE INDEX candles__timeframe__symbol_mts ON candles(_timeframe, _symbol, mts)`